tests/ci/create-osbuild.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+# following steps as noted on https://hackmd.io/ZbUOkeIXTjaP7XVpTSsrNw?view
+
+# Get OS data.
+source /etc/os-release
+
+ID=${ID:-}
+ARCH=$(arch)
+OS_VARIANT=${OS_VARIANT:-}
+IMAGE_TYPE=${IMAGE_TYPE:-}
+UUID=${UUID:-local}
+BOOT_LOCATION="http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/"
+
+
+# install osbuild and osbuild-tools, which contains osbuild-mpp utility
+dnf -y copr enable @osbuild/osbuild
+SEARCH_PATTERN='baseurl=https://download.copr.fedorainfracloud.org/results/@osbuild/osbuild/epel-8-\$basearch/'
+REPLACE_PATTERN='baseurl=https://download.copr.fedorainfracloud.org/results/@osbuild/osbuild/centos-stream-8-$basearch/'
+sed -i -e "s|$SEARCH_PATTERN|$REPLACE_PATTERN|" \
+	/etc/yum.repos.d/_copr\:copr.fedorainfracloud.org\:group_osbuild\:osbuild.repo
```

Comment:
Just to give you some context, by default, that copr repo has repos for **CentOS 8** and **CentOS Stream 8** (CS(), but by default, it enable the former.
The `sed` is to be sure the system uses the one from CS8, so it can build the install the Neptune app in the CS8 image we're building.

This works a bit different than the `osbuild-composer`. With the composer we indicate the repos (`sources`) to be used, but the `osbuild` uses the repos enabled in the host. That's why we need to enable them at this point, at the host.

We'll need to see how to do it when we try to build different OS versions (RHEL, C8, CS8, Fedora...), as we'll probably need to enable their repos to the host... :thinking: